MAUI/DataForm/Editing.md

```diff
@@ -0,0 +1,489 @@
+
+## Commit mode
+
+The [CommitMode] determines when the value should be committed to the data object.
+
+The supported commit modes are as follows:
+
+* LostFocus
+* PropertyChanged
+* Manual
+
+### LostFocus
+If the commit mode is LostFocus, the value is committed when the editor lost its focus.
+
+{% tabs %}
+{% highlight xaml %}
+<ContentPage  xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:DataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+             x:Class="CommitMode.MainPage">
+    <DataForm:SfDataForm
+        x:Name="DataForm"
+        CommitMode="LostFocus">
+    </DataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# %}
+this.DataForm.CommitMode = DataFormCommitMode.LostFocus;
+{% endhighlight %}
+{% endtabs %}
+
+
+### PropertyChanged
+If the commit mode is PropertyChanged, the value will be committed immediately when it is changed.
+
+{% tabs %}
+{% highlight xaml %}
+<ContentPage  xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:DataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+             x:Class="CommitMode.MainPage">
+    <DataForm:SfDataForm
+        x:Name="DataForm"
+        CommitMode="PropertyChanged">
+    </DataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# %}
+this.DataForm.CommitMode = DataFormCommitMode.PropertyChanged;
+{% endhighlight %}
+{% endtabs %}
+
+### Manual
+If the commit mode is Manual, the value should be committed manually by calling the [SfDataForm.Commit] method.
+
+{% tabs %}
+{% highlight xaml %}
+<ContentPage  xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:DataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+             x:Class="CommitMode.MainPage">
+    <DataForm:SfDataForm
+        x:Name="DataForm"
+        CommitMode="Manual">
+    </DataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# %}
+this.DataForm.CommitMode = DataFormCommitMode.Manual;
+{% endhighlight %}
+{% endtabs %}
+
+The following code commits the value of all the properties in the data object:
+
+{% tabs %}
+{% highlight c# %}
+this.DataForm.Commit();
+{% endhighlight %}
+{% endtabs %}
+
+## Converter
+
+To show the original value in different format or as different value, use the [DataFormValueConverter] attribute.
+
+### Changing original value of the DataForm property value using converter
+
+Here, the original value is multiplied by 10 and shown in editor. While committing, it is divided by 10 and stored in the data object.
+
+{% tabs %}
+{% highlight c# %}
+public class ValueConverterExt : IValueConverter
+{
+    public object Convert(object value)
```

Comment:
Convert and ConvertBack methods parameters are wrong, use proper method implementation. Always test code before using in UG